src/Symfony/Component/Workflow/Definition.php

```diff
@@ -17,8 +17,9 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-class Definition
+final class Definition
```

Comment:
Because then there is neither a way how to switch nor extend implementation.

https://ocramius.github.io/blog/when-to-declare-classes-final/